frontend/components/common/AppProviders.tsx

```diff
@@ -64,17 +65,19 @@ export function AppProviders({
 
    return (
       <AppProvider ifixitOrigin={ifixitOrigin ?? IFIXIT_ORIGIN}>
```

Comment:
Since we have a new context for the ifixit origin, I think we should remove this value from the AppProvider. Thoughts?

Reply:
Hmm, AppProvider only contains ifixitOrigin, so now I'm questioning why we split it out into its own package. The original goal is to remove packages with tons of unneeded dependencies from newsletter-sdk, but app doesn't seem to have a bunch of extra dependencies. Let me ping Andrew to clarify.

Reply:
Force pushed them out :+1: